v1/linkml-schemas/address.yaml

```diff
@@ -36,15 +37,28 @@ classes:
 
 slots:
   uprn:
+    slot_uri: gml:uprn
```

Comment:
I still can't see how these resolve to anything relative to the `gml` definition given above, I can buy that they actually don't, but if so then we should put a link into some documentation that defines the URIs.

Reply:
Yeah had a further look after we discussed the other day.  I need to look again.

Reply:
```suggestion
    slot_uri: adb:uprn
```

Reply:
added into new file commit